### DISCUSSION TITLE
Chia tokens have not come to the wallet balance

### DISCUSSION THREAD

**Discussion by Adel-Mingaleev** [Chia-Network/chia-blockchain#14379] [Support]
Hello. I downloaded the wallet from your official website. Then I transferred the Chia coins from the Huobi to my wallet, but they never came.  Transaction was successful, the address of the wallet is correct.
Please help me solve this problem.

Transaction:
https://xchscan.com/ru/txns/0x53b3685a5913d98882289a62faf799c6d52f77b41eeeb9e16e326f99ba36e5df

**Comment by Bodman1** [comment]
this transaction was made on January 18, 2023, 7:22 am, at 33 chia, that evening I also transferred 27.9482 chia kucoin from the exchange to my wallet, in the support of the exchange they wrote to me that the coins were sent to the wallet, but on the wallet I didn't see them. transaction data

The withdrawal you made on 2023-01-19 03:04:26 (UTC+08:00) was successful.
Token: | XCH
Amount: | 27.9482
Address: | xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72

**Reply by Adel-Mingaleev** [reply]
I had problem syncing my wallet with the network. I installed version 1.6.2 and it helped me.

**Reply by Bodman1** [reply]
didn't help me

**Comment by androiddevkotlin** [comment]
The same problem
